_unittest/test_28_Maxwell3D.py

```diff
@@ -75,6 +75,9 @@ def test_01_create_primitive(self):
         assert plate.solve_inside
         assert plate.material_name == "aluminum"
 
+        assert self.aedtapp.assign_material(plate, "pec")
```

Comment:
@IreneWoyna other than asserting if the method is successful, it's better to assert if the property solve inside is now set to the expected valuen

Reply:
sure, makes sense, I changed it:

self.aedtapp.assign_material(plate, "pec")
assert not plate.solve_inside